app/vep/models/pipeline_model.py

```diff
@@ -63,7 +63,7 @@ def create_config_ini_file(self):
 '''
 
     try:
-      with tempfile.NamedTemporaryFile(prefix="vep_", suffix=".ini", dir=UPLOAD_DIRECTORY, delete=False) as ini_file:
+      with tempfile.NamedTemporaryFile(prefix="vep_", suffix=".ini", dir=dir, delete=False) as ini_file:
```

Comment:
Do we need a NamedTemporaryFile anymore or can we go with a simple  config.ini file in temp dir?

Reply:
I'd also prefer simple `config.ini` here.